features/object-view-box.yml

```diff
@@ -0,0 +1,5 @@
+name: object-view-box
+description: The `object-view-box` CSS property defines the subset within an image that should be drawn in the content box of the image.
+spec: https://drafts.csswg.org/css-images-5/#the-object-view-box
+compat_features:
+  - css.properties.object-view-box
```

Comment:
Should the descendant key `css.properties.object-view-box.none` be a part of this? If not, why not?

Reply:
`none` is the initial value of the property and does nothing. I would expect any browser that supports `object-view-box` to also support the `none` value, from the get go.

I think it would be more interesting to add the `inset` and `rect` value types once they get added to BCD (I'll take a look and open a PR in BCD for it).

Reply:
OK, that's fine. I suspect we may end up tagging `none` into this feature as well (and the other values, as they get added) as we merge the tools to consume BCD more comprehensively, but this is fine for now. Thank you!